action.yml

```diff
@@ -26,17 +26,21 @@ runs:
   steps:
     - name: GHA setup-python for GH hosted runner
       id: gha-setup-python
-      if: inputs.is-self-hosted != 'true'
+      # We want to run this on anything but on self-hosted macs.
+      if: inputs.is-self-hosted == 'false' || runner.os != 'macOS'
```

Comment:
non-blocking: but does this `is-self-hosted` no longer make sense? Should we just make the parameter more specific to just self hosted macOS?

Reply:
It's useful, we want to run this step on hosted macOS. This is important that it keeps working in case of catastrophe in macstadium.